sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/AesKw.cs

```diff
@@ -0,0 +1,422 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    /// <summary>
+    /// AES Key Wrap algoritm as defined in https://tools.ietf.org/html/rfc3394
+    /// </summary>
+    internal abstract class AesKw
+    {
+        private const int BlockSizeInBits = 64;
+        private const int BlockSizeInBytes = BlockSizeInBits >> 3;
+
+        private static readonly byte[] _defaultIv = new byte[] { 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6 };
+
+        protected static RandomNumberGenerator Rng = RandomNumberGenerator.Create();
+
+        internal static Aes Create(byte[] key)
+        {
+            var aes = Aes.Create();
+
+            aes.Mode = CipherMode.ECB;
+            aes.Padding = PaddingMode.None;
+            aes.KeySize = key.Length * 8;
```

Comment:
Nit: I'd use bit-shift operator or multiplication consistently throughout code.

Reply:
This code is ported from directly track 1 and I'm leaving all the code in this class as is, since it's already been security reviewed for it's crypto implementation.